### DISCUSSION TITLE
Search Filter WebPart | AAD Properties

### DISCUSSION THREAD

**Discussion by Gryphos87** [microsoft-search/pnp-modern-search#2431] [Q&A]
Hello 🖖,

Is it possible to make use of AAD properties in the Search Filter WebPart? We use the SharePoint Search Data source in the Search Results WebPart. The context is to create a people search application.

Cheers

**Comment by wobba** [comment]
No easy way for this as sharepoint API and people search rely on the UPA profiles.
